nss-day-cohort-29/nutshell-midnight-dream
# NH pull request, HTML and button for task list

# Description

Coded the create and append functionality for the task list. Additionally created a button which will eventually open a "new task" form. Lastly, ran a text to ensure fetch works with the JSON to populate the existing task list when the page is loaded.


## Type of change

Please delete options that are not relevant.

- [ x] New feature (non-breaking change which adds functionality)


# Testing Instructions for Change Made

```git fetch --all
git checkout nh-form-createand-apend
```

1) ensure grunt is running
2)review page for title, taskName and TaskDate, followed by a new task button.
3) new task button with note "click" in the console.



# Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings
- [ x] I have added test instructions that prove my fix is effective or that my feature works
